components/launcher/src/lib.rs

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #[macro_use]
+extern crate habitat_common as common;
```

Comment:
In general, we'd like to move away from this pattern since these `extern crate` statements are generally no longer needed with Rust 2018 Edition.

Instead, each module just directly does a `use habitat_common`.

This also means that any `use crate::common::{...};`  statements now become `use habitat_common::{...};`. This is nice because it will mean that anything in `use crate::{...};` will actually be things in the current crate; everything else is going to then be treated like the external crate it is.

Reply:
I thought we still needed to do this if the crate had macros we are using. no?

Reply:
After a bit of googling its not clear how we would get away from the `extern crate` when needing to use macros and it seems we have maintained this pattern for the core crate for this reason if I am interpreting things correctly.

Reply:
never mind. I got it.